File: DSA/DSA Using Go/Linear_Search.go
package main

import "fmt"

func findIndex(sl []int, element int) int {
	index := 0
	for index < len(sl) {
		if sl[index] == element {
			return index
		}
		index++
	}
	return -1
}

func main() {
	var sl = []int{23, 45, 56, 14, 52}
	op := findIndex(sl, 90)
	fmt.Println(op)
}

File: DSA/DSA Using Python/Linear_Search.py
li = [2,6,3,7,4,9,1]

def findelement(li, element):
    index = 0
    while index < len(li):
        if li[index]==element:
            return index
        index+=1
    return -1

op = findelement(li, 78)
print(op)